src/ServiceProviders/RestAPI.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
@JasonTheAdams  Are we loading only report rest API endpoint if yes, do think name of this class can be different like `ReportRestAPI`

Reply:
We're currently only loading the Reports endpoints, but it's not actually specific to that. As more endpoints are built out they can use this same provider.